sql/src/main/java/io/crate/planner/consumer/ESGetStatementPlanner.java

```diff
@@ -40,9 +40,14 @@ public static Plan convert(QueriedDocTable table, Planner.Context context) {
                 || !table.querySpec().where().docKeys().isPresent());
 
         DocTableInfo tableInfo = table.tableRelation().tableInfo();
-        if(table.querySpec().where().docKeys().get().withVersions()){
+        if (table.querySpec().where().docKeys().get().withVersions()){
```

Comment:
would make sense to store table.querySpec().where().docKeys().get() in a local var